--- xca-0.3.1/Makefile.in.org	Wed Mar 19 15:50:44 2003
+++ xca-0.3.1/Makefile.in	Fri Apr 25 20:23:32 2003
@@ -12,7 +12,8 @@
 CC=@CC@
 
 CPPFLAGS=@CPPFLAGS@ @DEFS@ 
-CFLAGS=-Wall @ac_DEBUG@
+CFLAGS=@CFLAGS@ -Wall @ac_DEBUG@
+CXXFLAGS=@CXXFLAGS@ -Wall @ac_DEBUG@
 
 LDFLAGS=-Llib @LDFLAGS@
 LIBS=@LIBS@ -lxcadb -lpki
@@ -46,7 +47,7 @@
 $(OBJS): $(UI_H)
 
 %.o: %.cpp
-	$(CC) $(CPPFLAGS) $(CFLAGS) -c $< -o $@
+	$(CC) $(CPPFLAGS) $(CXXFLAGS) -c $< -o $@
 
 xca: $(OBJS) lib/libxcadb.a lib/libpki.a
 	$(CC) $(LDFLAGS) $(OBJS) $(LIBS) -o xca
@@ -78,11 +79,10 @@
 	#rm -rf ../$(TARGET)
 	
 install: xca
-	strip xca
-	install -m 755 -o root -d $(prefix)@prefix@/share/xca $(prefix)@prefix@/bin
-	install -m 755 -o root xca $(prefix)@prefix@/bin
-	install -m 644 -o root img/*.png $(prefix)@prefix@/share/xca
-	install -m 644 -o root xca_??.qm $(prefix)@prefix@/share/xca
+	install -m 755 -d $(prefix)@prefix@/share/xca $(prefix)@prefix@/bin
+	install -m 755 xca $(prefix)@prefix@/bin
+	install -m 644 img/*.png $(prefix)@prefix@/share/xca
+	install -m 644 xca_??.qm $(prefix)@prefix@/share/xca
 	
 
 moc_%.cpp: %.h %.cpp
