--- xca-0.6.4/configure	2008-03-11 10:03:35.000000000 +0100
+++ xca-0.6.4.orig/configure	2008-03-11 10:03:49.000000000 +0100
@@ -105,7 +105,7 @@
   for dir in ${DIRS};  do
     for dbn in $@; do
       for suffix in so dylib obj a; do
-        for lib in lib out; do
+        for lib in lib lib64 out; do
           if test -r "${dir}/${lib}/lib${dbn}.${suffix}"; then
             add_lib "${dir}/${lib}" "${dbn}" "${suffix}"
 	    echo "Found: lib${dbn}.${suffix} at ${dir}/${lib}"
