diff -urNp xca-0.5.1.org/widgets/MainWindow.cpp xca-0.5.1/widgets/MainWindow.cpp
--- xca-0.5.1.org/widgets/MainWindow.cpp	2004-06-13 23:22:37.000000000 +0200
+++ xca-0.5.1/widgets/MainWindow.cpp	2006-10-16 04:50:07.000000000 +0200
@@ -445,7 +445,7 @@ void MainWindow::Error(errorEx &err)
 	}
 }
 
-void MainWindow::dberr(const char *errpfx, char *msg)
+void MainWindow::dberr(const DbEnv *dbenv, const char *errpfx, const char *msg)
 {
 	errorEx e(QString(errpfx) + "\n" + msg);
 	Error(e);
diff -urNp xca-0.5.1.org/widgets/MainWindow.h xca-0.5.1/widgets/MainWindow.h
--- xca-0.5.1.org/widgets/MainWindow.h	2004-04-15 01:36:56.000000000 +0200
+++ xca-0.5.1/widgets/MainWindow.h	2006-10-16 04:49:48.000000000 +0200
@@ -104,7 +104,7 @@ class MainWindow: public MainWindow_UI
 	int initPass();
 	static int passRead(char *buf, int size, int rwflag, void *userdata);
 	static int passWrite(char *buf, int size, int rwflag, void *userdata);
-	static void dberr(const char *errpfx, char *msg);
+	static void dberr(const DbEnv *dbenv, const char *errpfx, const char *msg);
 	static NewX509 *newX509();
 	static QString md5passwd(const char *pass);
 	static void Error(errorEx &err);
