--- xca-0.4.2/Makefile.in.orig	Tue Jul  8 23:24:41 2003
+++ xca-0.4.2/Makefile.in	Tue Jul 22 17:42:33 2003
@@ -12,7 +12,8 @@
 CC=@CC@
 
 CPPFLAGS=-I. @CPPFLAGS@ @DEFS@ 
-CFLAGS=@ac_CC_WARN@ @ac_DEBUG@
+CFLAGS=@CFLAGS@ @ac_CC_WARN@ @ac_DEBUG@
+CXXFLAGS=@CXXFLAGS@ @ac_CC_WARN@ @ac_DEBUG@
 
 LDFLAGS=-Llib @LDFLAGS@
 LIBS=@LIBS@ -lpki
@@ -30,7 +31,7 @@
 re: clean all
 
 %.o: %.cpp
-	$(CC) $(CPPFLAGS) $(CFLAGS) -c $< -o $@
+	$(CC) $(CPPFLAGS) $(CXXFLAGS) -c $< -o $@
 
 xca: $(OBJS) lib/libpki.a $(OBJS)
 	$(CC) $(LDFLAGS) $(OBJS) $(LIBS) -o xca
