diff -uNdr xca-0.4.6.old/configure xca-0.4.6/configure
--- xca-0.4.6.old/configure	2003-10-13 11:01:21.000000000 +0200
+++ xca-0.4.6/configure	2003-12-20 18:11:53.000000000 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh -e
+#!/bin/bash -e
 
 #####################################################
 # this configure-script searches for the needed includes
diff -uNdr xca-0.4.6.old/widgets/MainWindow.cpp xca-0.4.6/widgets/MainWindow.cpp
--- xca-0.4.6.old/widgets/MainWindow.cpp	2003-11-25 20:34:02.000000000 +0100
+++ xca-0.4.6/widgets/MainWindow.cpp	2003-12-20 18:12:46.000000000 +0100
@@ -282,16 +282,16 @@
 
 void MainWindow::cmd_help(const char* msg) {
 
-printf(" -v show version information and exit
- -k expect all following non-option arguments to be RSA keys
- -r expect all following non-option arguments to be
-    Certificate signing requests or SPKAC requests
- -c expect all following non-option arguments to be Certificates
- -p expect all following non-option arguments to be PKCS#12 files
- -7 expect all following non-option arguments to be PKCS#7 files
- -l expect all following non-option arguments to be Revokation lists
- -t expect all following non-option arguments to be XCA templates
- -x Exit after processing all commandline options");
+printf(" -v show version information and exit\n"
+" -k expect all following non-option arguments to be RSA keys\n"
+" -r expect all following non-option arguments to be\n"
+"    Certificate signing requests or SPKAC requests\n"
+" -c expect all following non-option arguments to be Certificates\n"
+" -p expect all following non-option arguments to be PKCS#12 files\n"
+" -7 expect all following non-option arguments to be PKCS#7 files\n"
+" -l expect all following non-option arguments to be Revokation lists\n"
+" -t expect all following non-option arguments to be XCA templates\n"
+" -x Exit after processing all commandline options\n");
 
 qFatal("Cmdline Error (%s)\n", msg);
 }
